packages/venia-concept/src/components/Head/title.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { Title as _Title } from 'react-head';
+
+export default props => (
```

Comment:
For consistency in the codebase please also define and export this function separately.